packages/service/README.md

```diff
@@ -334,15 +342,76 @@ interface RgbppApiCkbTransactionHash {
   txhash: string;
 }
 
+interface RgbppApiTransactionStateParams {
+  with_data?: boolean;
+}
+
 interface RgbppApiTransactionState {
   state: RgbppTransactionState;
+  attempts: number;
+  failedReason?: string;
+  data?: {
+    txid: string;
+    ckbVirtualResult: {
+      ckbRawTx: CKBComponents.RawTransaction;
+      needPaymasterCell: boolean;
+      sumInputsCapacity: string;
+      commitment: string;
+    };
+  };
+}
+
+interface RgbppCell extends Cell {
+  typeHash?: Hash;
 }
 
 interface RgbppApiAssetsByAddressParams {
   type_script?: string;
   no_cache?: boolean;
 }
 
+interface RgbppApiBalanceByAddressParams {
+  type_script?: string;
+  no_cache?: boolean;
+}
+interface RgbppApiBalance {
+  address: string;
+  xudt: RgbppApiXudtBalance[];
+}
+interface RgbppApiXudtBalance {
+  name: string;
+  decimal: number;
+  symbol: string;
+  total_amount: string;
+  available_amount: string;
+  pending_amount: string;
+  type_hash: string;
+  type_script: Script;
+}
+
+interface RgbppApiActivityByAddressParams {
+  rgbpp_only?: boolean;
+  type_script?: string;
+  after_btc_txid?: string;
+}
+interface RgbppApiActivity {
+  address: string;
+  cursor: string;
+  txs: {
+    btcTx: BtcApiTransaction;
+    isRgbpp: boolean;
+    isomorphicTx?: {
+      ckbRawTx?: CKBComponents.RawTransaction;
```

Comment:
According to the [API document](https://www.notion.so/Assets-API-Activity-JoyID-d4a326cf6ddc47da806c72f5aa5852d9), it is recommended renaming the `ckbRawTx` to `ckbVirtualTx`.

Reply:
Updated at aecd837